src/Microsoft.Restier.WebApi/RestierController.cs

```diff
@@ -76,15 +76,11 @@ public IApi Api
             }
 
             IQueryable queryable = this.GetQuery();
-            QueryRequest queryRequest = new QueryRequest(queryable, this.includeTotalCount)
+            QueryRequest queryRequest = new QueryRequest(queryable, false)
             {
                 ShouldReturnCount = this.shouldReturnCount
             };
             QueryResult queryResult = await Api.QueryAsync(queryRequest, cancellationToken);
```

Comment:
There maybe a race condition here? When queryAsync is executed, the context QueryIncludeTotalCount is not set yet, and cause QueryExecutorWebApi does not call count query?